src/dotnet-grpc/Commands/CommandBase.cs

```diff
@@ -161,7 +161,7 @@ public void AddProtobufReference(Services services, string additionalImportDirs,
                 // If file is outside of the project, display the file under Protos/ directory
                 if (!Path.GetFullPath(resolvedPath).StartsWith(Project.DirectoryPath, StringComparison.OrdinalIgnoreCase))
                 {
-                    newItem.Xml.AddMetadata(LinkElement, Path.Combine(ProtosFolder, Path.GetFileName(file)!));
+                    newItem.Xml.AddMetadata(LinkElement, Path.Combine(ProtosFolder, Path.GetFileName(file)!), expressAsAttribute: true);
```

Comment:
Link attributes seem relatively short and it seems more compact to express it as an attribute instead.